### ISSUE TITLE
ClamAV Integration

### ISSUE THREAD

**Issue by henningwerner** [mail-in-a-box/mailinabox#858]
What about an Antivirus integration?

**Comment by gitacticia** [comment]
clamav can run safely in parallel. MiaB is ran over ubuntu. Just install and configure clamav on your box.

**Comment by ChiefGyk** [comment]
While it can be run, tutorials however have ended up mysteriously having my emails disappear. So what is the best way to set it up with an Ubuntu +MiaB system?

**Comment by gitacticia** [comment]
@ChiefGyk in order to install  type `sudo apt-get update && sudo apt-get install clamav` then cron a job to update viruses definitions with `sudo freshclam`, cron another to scan with _at least_ `sudo clamscan -r /home/` and tune with [an easy to google tutorial](https://www.howtoforge.com/tutorial/clamav-ubuntu/#clamav)

I think @henningwerner was wonderring about _**integration**_ within the admin console.
In term of security, I disagree with _integrated solutions_.

**Comment by ChiefGyk** [comment]
I am referring to it scanning email automatically inbound and outbound such as clamsmtp for a real email virus solution, rather than waiting for it to scan for viruses.

**Comment by gitacticia** [comment]
Ok sorry. I have written a little python script that **trhew** all _new_ or _modified_ files (detected with a fast hash lookup table on an unjournalized FS, and with the modified tag on ext3). That was pretty efficient but cpu-costly.

Is it the way you meant "scanning automatically" ?

**Comment by ChiefGyk** [comment]
Yes, that's what I am referring to. However when I referred to a tutorial in mailinabox discussion, it lead to my emails vanishing from the outbox during testing. I was hoping MiaB would have some solution so my Windows clients have some piece of mind. I personally use GNU/Linux only, but my mom and dad who also use my email server I made (it's a family email server) are Windows only people. I am sure many MiaB users are in a similar situation and would like to see some way of AV integration
